e2e2/test/images/nvidia/Dockerfile

```diff
@@ -1,10 +1,15 @@
 # Start with the NVIDIA CUDA base image
 FROM nvidia/cuda:12.5.0-devel-ubuntu22.04
 
-ARG EFA_INSTALLER_VERSION=latest
+# Add ENV to make ARG values available at runtime
+ARG EFA_INSTALLER_VERSION=1.34.0
+ARG NCCL_VERSION=2.18.5
```

Comment:
Why are we using an older version of nccl here ? General recommendation is to use either last or last-1 release here